config/brakeman.ignore

```diff
@@ -0,0 +1,146 @@
+{
+  "ignored_warnings": [
+    {
+      "warning_type": "Unscoped Find",
+      "warning_code": 82,
+      "fingerprint": "07081676ffe0bf0ef753045ae9dcc86c848df5f35a9e48a22692159d5352fbd1",
+      "check_name": "UnscopedFind",
+      "message": "Unscoped call to `ProviderRelationshipPermissions#find`",
+      "file": "app/controllers/provider_interface/provider_relationship_permissions_controller.rb",
+      "line": 32,
```

Comment:
What happens to this if the code gets moved around? Can you try it locally by shifting the `def` a few lines and see how Brakeman reacts, does it keep this file updated for us or is it a bit of a pain?

Reply:
It looks pretty robust. I swapped two methods around and it was able to keep up. I think it actually looks at the `location` and `code` keys rather that the `line`. 

```    {
      "warning_type": "Unscoped Find",
      "warning_code": 82,
      "fingerprint": "4c106936481ffa09c60383aed2a7b8e931b1672602d3f5a69ccaf9bd688cbf62",
      "check_name": "UnscopedFind",
      "message": "Unscoped call to `ProviderRelationshipPermissions#find`",
      "file": "app/controllers/provider_interface/provider_relationship_permissions_setup_controller.rb",
      "line": 40,
      "link": "https://brakemanscanner.org/docs/warning_types/unscoped_find/",
      "code": "ProviderRelationshipPermissions.find(params[:id])",
      "render_path": null,
      "location": {
        "type": "method",
        "class": "ProviderInterface::ProviderRelationshipPermissionsSetupController",
        "method": "save_permissions"
      },
      "user_input": "params[:id]",
      "confidence": "Weak",
      "note": ""
    },